README.md

```diff
@@ -281,11 +297,9 @@ decisions.
 
 Does that make reading about these details a waste of time?
 Not at all. Important decisions rarely should or can be blindly delegated.
-This Guide *complements but does not replace* the advice you need get from professionals.
-You make the best decisions when you understand things as well as have the support of
-professionals.
-Even experienced attorneys and tax experts can say things that are confusing to clients,
-and it is far easier to work with them when you have deeper understanding yourself.
+This Guide *complements but does not replace* the advice you get from professionals.
+Working with the support of a professional is most helpful when you have an understanding
```

Comment:
This now omits the point about making better decisions as a result of knowledge plus advice, which I believe is key.

"helpful" -> "easier and more effective"

Reply:
Fixed ("...the support of a professional can help you make better decisions when you have...")